test/KillSwitchOracleIntegration.t.sol

```diff
@@ -0,0 +1,285 @@
+
+        _assertReserve({
+            asset:                DAI,
+            active:               true,
+            frozen:               true,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 1,
+            liquidationBonus:     104_50
+        });
+        _assertReserve({
+            asset:                SDAI,
+            active:               true,
+            frozen:               false,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 76_00,
+            liquidationBonus:     104_50
+        });
+        _assertReserve({
+            asset:                USDC,
+            active:               true,
+            frozen:               true,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 0,
+            liquidationBonus:     0
+        });
+        _assertReserve({
+            asset:                WETH,
+            active:               true,
+            frozen:               false,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 82_50,
+            liquidationBonus:     105_00
+        });
+        _assertReserve({
+            asset:                WSTETH,
+            active:               true,
+            frozen:               false,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 79_50,
+            liquidationBonus:     107_00
+        });
+        _assertReserve({
+            asset:                WBTC,
+            active:               true,
+            frozen:               false,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 75_00,
+            liquidationBonus:     107_00
+        });
+        _assertReserve({
+            asset:                GNO,
+            active:               true,
+            frozen:               true,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 25_00,
+            liquidationBonus:     110_00
+        });
+        _assertReserve({
+            asset:                RETH,
+            active:               true,
+            frozen:               false,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 79_50,
+            liquidationBonus:     107_00
+        });
+        _assertReserve({
+            asset:                USDT,
+            active:               true,
+            frozen:               true,
+            paused:               false,
+            ltv:                  0,
+            liquidationThreshold: 0,
+            liquidationBonus:     0
+        });
+    }
+
+    function _assertReserve(
```

Comment:
This declaration is doubled. I'd create a test base with these utils inside.

Reply:
I think it's fine. A small amount of duplication is fine, and probably more overhead to do a base contract for just this one thing.